plugins/catalog-backend/migrations/20210302150147_refresh_state.js

```diff
@@ -185,13 +185,18 @@ exports.up = async function up(knex) {
       .string('key')
       .notNullable()
       .comment('A key that occurs in the entity');
+    table
+      .string('original_value')
+      .nullable()
+      .comment('Holds the corresponding original case sensitive value');
```

Comment:
A note here that you can't modify existing migrations. These have already been run in the past, on existing installations. You will need to make a new integration, which is functional on top of everything else. You'll see a lot of examples of alter table statements and similar in the other migration files in here, if you want some ideas and inspiration.

Reply:
Thanks, I did not know that. Will make the necessary changes.

Reply:
@freben I have a question about creating a migration file. Is there a knex command to do that? Or I simply create one with the timestamp? Any documentation on this would be really helpful.

Reply:
Sorry we were absent last week. Yeah just copy-paste by hand for now, as mentioned on Discord.